### PR TITLE
Search 7.2 | Post-Release Improvements | ES7/LTR | LRDOCS-7450

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-7450, Let me know if you think there are more updates/corrections to make.

### CODE REVIEW

File: en/deployment/articles-dxp/03-installing-a-search-engine/03-enterprise-search/03-ltr/01-learning-to-rank-intro.markdown [hunk 1]
```diff
@@ -59,9 +59,9 @@ queries sent to Elasticsearch:
   required, with the appropriate Elasticsearch Connector version installed.
 
 - A [Liferay Enterprise Search](https://help.liferay.com/hc/en-us/articles/360014400932) 
-  (LES) is required for Learning to Rank. Once you have a subscription, 
-  [download LES and install the LPKG files](/docs/7-2/user/-/knowledge_base/u/installing-apps-manually#installing-apps-manually)
-  to your @product@ server.
+  (LES) subscription is required for Learning to Rank. Once you have a
+  subscription, download the Liferay Connector to Learning to Rank LPKG file and
```
Comment (lipusz) [inline en/deployment/articles-dxp/03-installing-a-search-engine/03-enterprise-search/03-ltr/01-learning-to-rank-intro.markdown]: Download from https://customer.liferay.com/downloads

**Comment by lipusz** [comment]
**Started reviewing...** :mag::nerd_face:

**Comment by lipusz** [comment]
Hey Russel,

Another thing we discovered during the Tokyo sessions was that currently we say this in the Configuring Learning to Rank article:
https://help.liferay.com/hc/en-us/articles/360035444912-Configuring-Learning-to-Rank

>Before beginning, you must have a remote Elasticsearch 7.4 cluster communicating with Liferay DXP 7.2.

But there is no LTR plugin (currently) for 7.4.1: http://es-learn-to-rank.labs.o19s.com/

So instead of being specific maybe we should just say that that for LTR you need to be on an ES6 or ES7 server version that is supported by Liferay (link to the LES matrix) **AND** and the LTR plugin is also available.

Can you please make this change?

Thanks,
Tibi

/cc @jordirodo thx Jordi for catching it!

**Comment by lipusz** [comment]
The rest of the changes look good to me. :+1:

**Comment by rbohl** [comment]
Sent for publication: https://github.com/sez11a/liferay-docs/pull/4706

**Pull request closed without merging**